mlir/lib/Dialect/Rock/IR/RockDialect.cpp

```diff
@@ -519,12 +518,17 @@ GemmSize GemmSize::fromConvolution(ConvOpType type,
 }
 
 static LogicalResult verifyGemmTypes(Operation *op, GemmFeatures features,
-                                     Type elemTypeA, Type elemTypeB,
-                                     Type elemTypeC) {
+                                     StringRef arch, Type elemTypeA,
+                                     Type elemTypeB, Type elemTypeC) {
+  bool isGfx12 = arch.contains("gfx12");
```

Comment:
Let's make this a test for gfx11 instead? Since future chips should have fp8 support